src/utilities/mocks/fullprofile.js

```diff
@@ -0,0 +1,1125 @@
+export const fullprofileMockData = {
+  contact: {
+    birthDate: '1979-08-14',
+    preferredPhoneType: 'Mobile',
+    preferredAddress: 'Home',
+    preferredEmail: 'Home Email',
+    raceEthnicityFromRegistrar: null,
+    selfReportedEthnicity: null,
+    selfReportedEthnicityCustom: null,
+    maritalStatus: 'Married',
+    profilePhotoURL: null,
+    digitalNameOverrideFlag: false,
+    envelopeNameOverrideFlag: false,
+    name: {
+      digitalName: 'Teri Dactyl',
+      pronouns: 'He/him',
+      myFriendsCallMe: 'Teri',
+      envelopeName: 'Mr. Teri Dactyl',
+      username: 'zguan',
```

Comment:
```suggestion
      username: 'teridactyl',
```

Or do you need this as zguan for something?